app/views/users/show.html.erb

```diff
@@ -0,0 +1 @@
+<%= @user.username >
```

Comment:
Add new line (I'll merge to master and add it so that I can continue working on it)